### ISSUE TITLE
Button color suggestion does not display like we have in other blocks

### ISSUE THREAD

**Issue by Anjali29dots** [WordPress/gutenberg#48308]
### Description

The suggestion does not appear when we select odd colors for the visibility purpose.

### Step-by-step reproduction instructions

Go to add Button block.
Go to Block settings and select Style tab.
Go to select text and background color which are not visible.
The suggestion does not appear

### Screenshots, screen recording, code snippet

Actual Screenshot: https://share.cleanshot.com/JH6jCDgMQgkNsV935mW6
Expected Screenshot: https://share.cleanshot.com/pps6ygK42bd1C8mZs1w7

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ghost** [comment]
Submitted a PR for this #48313
